tests/sentry/tasks/test_sentry_apps.py

```diff
@@ -50,72 +49,29 @@ def __init__(self):
         self.body = "blah blah"
 
 
-class ResponseMock:
-    def __init__(self, headers, content, text, ok, status_code, request):
-        self.headers = headers
-        self.content = content
-        self.text = text
-        self.ok = ok
-        self.status_code = status_code
-        self.request = request
-
-    def get(self, request=None, content=None):
-        if request:
-            return self.request
-        if content:
-            return self.content
-        return None
-
-
 headers = {"Sentry-Hook-Error": "d5111da2c28645c5889d072017e3445d", "Sentry-Hook-Project": "1"}
+html_content = "a bunch of garbage HTML"
+json_content = '{"error": "bad request"}'
 
-mock_fail_resp = ResponseMock(
-    headers=headers,
-    content={},
-    text="",
-    ok=False,
-    status_code=400,
-    request=RequestMock(),
-)
-
-mock_fail_html_resp = deepcopy(mock_fail_resp)
-mock_fail_html_resp.content = "a bunch of garbage HTML"
-
-MockFailureHTMLContentResponseInstance = MagicMock(
-    status_code=mock_fail_html_resp.status_code,
-    headers=mock_fail_html_resp.headers,
-    content=mock_fail_html_resp.content,
-    request=mock_fail_html_resp.request,
+MockResponse = namedtuple(
```

Comment:
personally I like `typing.NamedTuple` -- but this is fine for now!  thanks for the fix \o/